sympy/vector/tests/test_vector.py

```diff
@@ -208,6 +218,26 @@ def test_vector_simplify():
     assert simplify(Vector.zero) == Vector.zero
 
 
+def test_vector_equals():
+    assert (2*i).equals(j) is False
+    assert i.equals(i) is True
+
+    # https://github.com/sympy/sympy/issues/25915
+    A = (sqrt(2) + sqrt(6)) / sqrt(sqrt(3) + 2)
+    assert (A*i).equals(2*i) is True
+    assert (A*i).equals(3*i) is False
+
+    # Test comparing vectors in different coordinate systems
+    D = C.orient_new_axis('D', pi/2, C.k)
+    assert (D.i).equals(C.j) is True
+    assert (D.i).equals(C.i) is False
+
+
+def test_vector_conjugate():
+    # https://github.com/sympy/sympy/issues/27094
+    assert (I*i + (1 + I)*j + 2*k).conjugate() == -I*i + (1 - I)*j + 2*k
```

Comment:
> But I think that tests for the issues should be added before confirming

There are tests for the linked issues here. Did you mean something else?

Reply:
Okay, I didn't notice the changes about this file. Sorry